### PR TITLE
Html rag 1 -- Crawl a website / process HTML / run RAG queries

### PR DESCRIPTION
**Pull request by sujee**
## Why are these changes needed?

This examples shows how to 
- crawl a website
- process HTML files
- run RAG query on HTML file contents

### CODE REVIEW

**Review by shahrokhDaijavad — APPROVED** [review]
@sujee I have now tested this example successfully by going through notebooks 1 to 4. Looks good.

**Review by touma-I — CHANGES_REQUESTED** [review]
Hi. Can the same thing be accomplished without having to checkin all the HTML content ?

**Comment by sujee** [comment]
> Hi. Can the same thing be accomplished without having to checkin all the HTML content ?

removed HTML input files that were checked in.  @touma-I

**Comment by sujee** [comment]
@touma-I please hold  off merging this until I move it to 0.2.2.  thx

**Comment by sujee** [comment]
@touma-I @shahrokhDaijavad 
updated to v0.2.3-dev1
good to go

**Comment by shahrokhDaijavad** [comment]
@sujee @touma-I I have now tested the latest version successfully, so from a functionality point of view, I approve this version.

**Review by shahrokhDaijavad — APPROVED** [review]

File: examples/notebooks/html-processing/requirements.txt
Comment (touma-I) [inline examples/notebooks/html-processing/requirements.txt:1]: Hi @sujee, question: Did you try to use both extra in the same line?
```
data-prep-toolkit-transforms[html2parquet,web2parquet]==0.2.3.dev1
```

**Review by touma-I — APPROVED** [review]

**Pull request merged**